lib/inspec/cli.rb

```diff
@@ -168,9 +168,70 @@ def archive(path)
     pretty_handle_exception(e)
   end
 
-  desc 'exec PATHS', 'run all test files at the specified PATH.'
+  desc 'exec LOCATIONS', 'run all test files at the specified LOCATIONS.'
   long_desc <<~EOT
-    Loads the given profile(s) and fetches their dependencies if needed.  Then connects to the target and executes any controls contained in the profiles.  One or more reporters are used to generate output.  If all tests passed (no fails, no skips) exit code 0 is returned.  If some tests skipped but none failed, exit code 101 is returned. If at least one test failed, exit code 100 is returned.  If inspec failed for any other reason, exit code 1 is returned.
+    Loads the given profile(s) and fetches their dependencies if needed. Then
+    connects to the target and executes any controls contained in the profiles.
+    One or more reporters are used to generate output. If all tests passed
+    (no fails, no skips) exit code 0 is returned. If some tests skipped but
+    none failed, exit code 101 is returned. If at least one test failed, exit
+    code 100 is returned. If inspec failed for any other reason, exit code 1
+    is returned.
+
+    Below are some examples of using `exec` with different test LOCATIONS:
+
+    Automate:
+      ```
+      inspec compliance login
+      inspec exec compliance://username/linux-baseline
+      ```
+
+    Supermarket:
+      ```
+      inspec exec supermarket://username/linux-baseline
+      ```
+
+    Local profile (executes all tests in `controls/`):
+      ```
+      inspec exec /path/to/profile
+      ```
+
+    Local single test (doesn't allow attributes or custom resources)
+      ```
+      inspec exec /path/to/a_test.rb
+      ```
+
+    Git via SSH (uses SSH keys if repo is private):
```

Comment:
Assuming this clones, valid SSH keys are required whether the repo is public or not:

![screen shot 2018-11-08 at 00 17 11](https://user-images.githubusercontent.com/156460/48179149-aea45f80-e2eb-11e8-904e-c363968837cc.png)

Reply:
Hmm...interesting...didn't know that. Will update the the comment.

Reply:
Hmm...interesting...didn't know that. Will update the comment.